dkist/dataset/tests/test_dataset.py

```diff
@@ -71,28 +72,49 @@ def test_dimensions(dataset, dataset_3d):
 
 
 def test_load_from_directory():
-    ds = Dataset.from_directory(os.path.join(rootdir, 'EIT'))
+    ds = load_dataset(os.path.join(rootdir, 'EIT'))
     assert isinstance(ds.data, da.Array)
     assert isinstance(ds.wcs, gwcs.WCS)
     assert_quantity_allclose(ds.dimensions, (11, 128, 128)*u.pix)
     assert ds.files.basepath == Path(os.path.join(rootdir, 'EIT'))
 
 
-def test_from_directory_no_asdf(tmpdir):
+def test_from_directory_no_asdf(tmp_path):
     with pytest.raises(ValueError) as e:
-        Dataset.from_directory(tmpdir)
+        load_dataset(tmp_path)
         assert "No asdf file found" in str(e)
 
 
 def test_from_not_directory():
     with pytest.raises(ValueError) as e:
-        Dataset.from_directory(rootdir / "notadirectory")
+        load_dataset(rootdir / "notadirectory")
         assert "directory argument" in str(e)
 
 
+def test_load_tiled_dataset():
+    ds = load_dataset(os.path.join(rootdir, 'test_tiled_dataset-1.0.0_dataset-1.1.0.asdf'))
+    assert ds.shape == (3, 3)
+
+
+def test_load_with_old_methods():
```

Comment:
These tests appear to be running and passing, but for some reason codecov claims that the contents of `from_directory` and `from_asdf` aren't getting hit. Not sure what that's about.